software/due-date-changes.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import re
+import sys
+import time
+
+from datetime import datetime
+from jira import JIRA
+
+
+def dump(obj):
+    for attr in dir(obj):
+        print("obj.%s = %r" % (attr, getattr(obj, attr)))
+
+
+def print_help(stream=sys.stderr):
+    help_msg = """Usage: {0} [--projects] LIST,OF,PROJECTS
+"""
+    stream.write(help_msg.format(sys.argv[0]))
```

Comment:
Alternatively,

```suggestion
    help_msg = """Usage: {0} [--projects] LIST,OF,PROJECTS"""
    print(help_msg.format(sys.argv[0]), file=stream)
```

Reply:
I'm going to remove this block, the print_help() function is boilerplate from a different script and never actually called here.

Reply:
It's still useful to have help/usage output, even for our metric scripts here!!

Although now that i look at it the right bracket belongs after the optional project list

ie,

```
Usage: {0} [--projects LIST,OF,PROJECTS]
```

Reply:
We still have help/usage output, it's provided by argparse. The full `print_help()` function was carried over from different code that works differently, we never called it here.

As for the brackets I'll fix that in a follow up commit.

Reply:
Don't mind @edquist he only knows the ways of `getopt` :)

Reply:
Agreed -- please, don't mind me!

It's true: if it's not getopt, i head pretty much straight for the hills.

And i conveniently ignore prominent counterexamples like find(1) when making blanket claims about unix norms.

You could also compromise and accept a single EM DASH for long options.  I mean, it's the decade 203 after all.